lib/constants/queryString.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * query strings for job table
+ */
+
+const JobQuery = {
+    GetRow: "select * from job where id =($1)",
```

Comment:
please follow a convention for the key names. You use no underscores here but you use underscores in the responseAndStatus.js file